updater/src/main/java/Main.java

```diff
@@ -225,46 +225,25 @@ default Package getPackageDetails(int version, String architecture) throws IOExc
     branches:
       - master
     paths:
-      - '.github/workflows/jdk{VERSION}.yml'
+      - '.github/workflows/reusable-cask-checks.yml'
       - 'Casks/zulu-jdk{VERSION}.rb'
   pull_request:
     branches:
       - master
     paths:
-      - '.github/workflows/jdk{VERSION}.yml'
+      - '.github/workflows/reusable-cask-checks.yml'
       - 'Casks/zulu-jdk{VERSION}.rb'
 
 jobs:
-
-  check:
-    name: Check
-    strategy:
-      matrix:
-        os:
-          # macos-latest is based on arm64.
-          - macos-latest
-          # macos-13 is based on x64.
-          - macos-13
-    runs-on: ${{ matrix.os }}
-    env:
-      HOMEBREW_COLOR: 1
-      HOMEBREW_DEVELOPER: 1
+  checkout:
+    runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v4
```

Comment:
This step is necessary for using `.github/workflows/reusable-cask-checks.yml` on the current branch.

Reply:
547df7687bf5f931826890c5ad32ac526c9f6fdf